src/main/java/com/minecolonies/coremod/entity/EntityCitizen.java

```diff
@@ -1535,14 +1535,19 @@ public DesiredActivity getDesiredActivity()
             return DesiredActivity.WORK;
         }
 
+        if (BarbarianUtils.getClosestBarbarianToEntity(this, DISTANCE_OF_ENTITY_AVOID) != null && BarbarianUtils.getClosestBarbarianToEntity(this, DISTANCE_OF_ENTITY_AVOID).isEntityAlive())
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Split this 189 characters long line (which is greater than 180 authorized). [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS00103)